WordPress/src/main/java/org/wordpress/android/util/config/RemoteConfig.kt

```diff
@@ -37,8 +45,11 @@ class RemoteConfig
     private suspend fun fetchRemoteFlags() {
         Log.e("Refreshing remote flags", " ")
         val response = featureFlagStore.fetchFeatureFlags(
-                deviceId = "12345",
-                identifier = BuildConfig.APPLICATION_ID
+                buildNumber = BuildConfig.VERSION_CODE.toString(),
+                deviceId = preferences.getString(WPCOM_PUSH_DEVICE_UUID, "")?:"",
+                identifier = BuildConfig.APPLICATION_ID,
```

Comment:
@AjeshRPai  - What's the difference between BuidConfig.APPLICATION_ID and packageName from package manager?

Reply:
As discussed over sync call the BuidConfig.APPLICATION_ID captures the entire package name with Flavour as well.